skrub/_utils.py

```diff
@@ -12,9 +12,10 @@
 
 
 class LRUDict:
-    """dict with limited capacity
```

Comment:
it doesn't matter this time but in future PRs try not to add changes that are not directly related to the PR's stated goal as it makes review harder

Reply:
I will ! Sorry about this, I opened another PR for cosmetic changes but was left with a tiny overlap in files related to this PR